set.mm

```diff
@@ -261972,23 +261969,22 @@ consisting of the same ring regarded as a (left) module over itself, see
 
   ${
     $d r i $.
-    $( The ` i ` -dimensional free module over a ring ` r ` is the product of
-       ` i ` -many copies of the ring with componentwise addition and
-       multiplication.  If ` i ` is infinite, the allowed vectors are
-       restricted to those with finitely many nonzero coordinates; this ensures
-       that the resulting module is actually spanned by its unit vectors.
-       (Contributed by Stefan O'Rear, 1-Feb-2015.) $)
+    $( Define the function associating with a ring and a set the direct sum
+       indexed by that set of copies of that ring regarded as a left module
+       over itself.  Recall from ~ df-dsmm that an element of a direct sum has
+       finitely many nonzero coordinates.  (Contributed by Stefan O'Rear,
+       1-Feb-2015.) $)
```

Comment:
From my point of view, to much information is removed by the new comment. The old and new version of the comment should be merged somehow.

Reply:
What was remove gives a wrong intuition. It talks about "the i-dimensional", but if $i$ is replaced by $\pi$, then one obtains the direct sum of uncountably many copies of the ring (simply because in set.mm, $\pi$ is encoded as some uncountable set). Then, there is no notion of dimension for general modules. I kept the last sentence recalling that elements of a direct sum have finitely many nonzero summands.